modules/Bio/EnsEMBL/Production/Pipeline/GPAD/FindFile.pm

```diff
@@ -0,0 +1,64 @@
+=head1 LICENSE
+
+Copyright [2009-2019] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=head1 NAME
+
+Bio::EnsEMBL::Production::Pipeline::GPAD::FindFile;
+
+=head1 DESCRIPTION
+
+=head1 AUTHOR
+
+maurel@ebi.ac.uk
+
+=cut
+package Bio::EnsEMBL::Production::Pipeline::GPAD::FindFile;
+
+use strict;
+use warnings;
+use Data::Dumper;
```

Comment:
Most of these libs aren't used, only Registry...

Reply:
Thanks, I will clean up.